public/app/features/admin/AdminFeatureTogglesTable.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { Switch } from '@grafana/ui';
+
+type FeatureToggle = {
+  name: string;
+  enabled: boolean;
+  description: string;
+};
+
+interface Props {
+  featureToggles: FeatureToggle[];
+}
+
+export function AdminFeatureTogglesTable({ featureToggles }: Props) {
+  return (
+    <table className="filter-table form-inline filter-table--hover">
```

Comment:
As we want to move away from using SASS styles, would it be possible to look into using a component that doesn't require a classname? I'll ask around and see if I find something.

Reply:
Have a look at this, it might support your use case https://developers.grafana.com/ui/latest/index.html?path=/story/experimental-interactivetable--basic

Reply:
thanks for the review, the interactivetable was perfect for our use case, I added the changes with the component